### PR TITLE
AWS Lambda Has Increased Memory Limits

### PR DESCRIPTION
**Pull request by ewascent**
AWS has announced a new memory limit for AWS Lambda. The Limit is now 10240 MB.

Screen Shot 2020-12-03 at 9 44 41 AM

### CODE REVIEW

File: troposphere/awslambda.py [hunk 1]
```diff
@@ -12,7 +18,7 @@
     'AWS_LAMBDA_FUNCTION_NAME',
     'AWS_LAMBDA_FUNCTION_VERSION',
     'AWS_LAMBDA_LOG_GROUP_NAME',
-    'AWS_LAMBDA_LOG_STREAM_NAME',
+    'AWS_LAMBDA_LOGbasestringEAM_NAME',
```
Comment (PatMyron) [inline troposphere/awslambda.py:21]: assuming this was accidental?
```suggestion
    'AWS_LAMBDA_LOG_STREAM_NAME',
```

Comment (ewascent) [reply troposphere/awslambda.py:21]: Uh..lemme fix that!

**Comment by ewascent** [comment]
I do not seem to have the ability to link this to issue 
https://github.com/cloudtools/troposphere/issues/1843

**Comment by adamdavis40208** [comment]
Can we please have 20480 while you're in there ;)

(j/k thanks for the contribution!)

**Comment by ewascent** [comment]
> Can we please have 20480 while you're in there ;)
> 
> (j/k thanks for the contribution!)

I mean we can...but AWS isn't ready for that level of awesome

**Comment by ewascent** [comment]
Linting failed on files unrelated to this pull request. 
```
./troposphere/validators.py:314:80: E501 line too long (86 > 79 characters)
                'REDHAT_ENTERPRISE_LINUX', 'SUSE', 'CENTOS', 'DEBIAN', 'ORACLE_LINUX']
                                                                               ^
./troposphere/kinesisanalyticsv2.py:10:80: E501 line too long (80 > 79 characters)
VALID_RUNTIME_ENVIRONMENTS = ('SQL-1_0', 'FLINK-1_6', 'FLINK-1_8', 'FLINK-1_11')
```

**Comment by PatMyron** [comment]
> Linting failed on files unrelated to this pull request.
> 
> ```
> ./troposphere/validators.py:314:80: E501 line too long (86 > 79 characters)
>                 'REDHAT_ENTERPRISE_LINUX', 'SUSE', 'CENTOS', 'DEBIAN', 'ORACLE_LINUX']
>                                                                                ^
> ./troposphere/kinesisanalyticsv2.py:10:80: E501 line too long (80 > 79 characters)
> VALID_RUNTIME_ENVIRONMENTS = ('SQL-1_0', 'FLINK-1_6', 'FLINK-1_8', 'FLINK-1_11')
> ```

Sorry, that was me since [Travis wasn't showing in PRs until this week](https://github.com/cloudtools/troposphere/pull/1790). Fixing here: https://github.com/cloudtools/troposphere/pull/1845

**Comment by ewascent** [comment]
@PatMyron was not trying to be lazy. I wanted to keep the scope of the changes clear since I am a random human to this project. :)

**Comment by markpeek** [comment]
Thanks! Further update likely we should move from a list of values to just checking the upper/lower bounds and alignment.

**Pull request merged**